.golangci.yml

```diff
@@ -282,6 +282,15 @@ issues:
   - linters:
       - staticcheck
     text: "SA1019: feature.ClusterResourceSet is deprecated: ClusterResourceSet feature is now GA and the corresponding feature flag will be removed in 1.12 release."
+  - linters:
+      - staticcheck
+    text: "SA5007: infinite recursive call"
```

Comment:
Is this still needed after the infinitie recursive call was removed? :)

Reply:
No